recipe/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  name: libqdldl
+  version: 0.1.8
+
+package:
+  name: ${{ name|lower }}
```

Comment:
Same request here as for qdldl-python: do not template over the name. It gives the 100% incorrect impression that this is a variable that can be changed, and needlessly makes the recipes harder to read through indirections.